decorator.rb

```diff
@@ -0,0 +1,29 @@
+require_relative './nameable'
+
+class Decorator < Nameable
+  attr_accessor :nameable
+
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    name_cap = @nameable.correct_name
+    name_cap.upcase
+  end
```

Comment:
- [ ] Kindly use the `capitalize` method instead of `upcase` as per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=that%20capitalizes%20the%20output%20of%20%40nameable.correct_name). `upcase` will make each character of the word capital, but, `capitalize` will make only the first character of the word capital.